tests/utils.cpp

```diff
@@ -222,6 +222,25 @@ TEST(utils, get_cgroup_path_in_hierarchy)
   }
 }
 
+TEST(utils, parse_kconfig)
+{
+  char path[] = "/tmp/configXXXXXX";
+  int fd = mkstemp(path);
```

Comment:
Might be nice to unlink this after use

Reply:
Done and resolved the changelog conflict.